### PR TITLE
MAGECLOUD-3760 service compatibility table

### PR DESCRIPTION
**Pull request by hguthrie**
This PR adds a service compatibility table to the services topic. I placed this at the bottom of the topic because there are significant changes to this topic currently set to merge to master very soon.

### CODE REVIEW

File: guides/v2.2/cloud/project/project-conf-files_services.md [hunk 1]
```diff
@@ -99,3 +99,17 @@ For example, the current default storage amount per project is 5GB, or 5120MB. Y
...
 
 For services to be available to an application in your project, you must specify [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) between applications and services in the `.magento.app.yaml` file.
+
+## Service versions available
+
+The following table lists the services used in {{site.data.var.ece}} and their version compatibility with the Magento application.
+
+Service   | Supported Versions
+--------- | -------------------
+`mariadb` | 10.0 to 10.2
```
Comment (oshmyheliuk) [inline guides/v2.2/cloud/project/project-conf-files_services.md]: elasticsearch is missed
```
        ServiceInterface::NAME_ELASTICSEARCH => [
            '<2.2.0' => '~1.7.0 || ~2.4.0',
            '>=2.2.0 <2.2.8 || 2.3.0' => '~1.7.0 || ~2.4.0 || ~5.2.0',
            '>=2.2.8 <2.3.0 || >=2.3.1' => '~1.7.0 || ~2.4.0 || ~5.2.0 || ~6.5.0',
        ],
```

Comment (hguthrie) [reply guides/v2.2/cloud/project/project-conf-files_services.md]: Ha! Good catch. Let me add that in. I'm not sure what happened...I think when I alphabetized them. Poor Elasticsearch got left out. I'm on it!

File: guides/v2.2/cloud/project/project-conf-files_services.md [hunk 2]
```diff
@@ -148,3 +148,18 @@ The current default storage amount per project is 5GB, or 5120MB. You can distri
...
+Service   |  Magento 2.3  | Magento 2.2
+--------- | ------------- | ------------
+`elasticsearch` | Magento 2.3.1 and later—1.7, 2.4, 5.2, 6.5<br>Magento 2.3.0—1.7, 2.4, 5.2 | Magento versions 2.2.8 and later—1.7, 2.4, 5.2, 6.5<br>Magento 2.2.0 to 2.2.7—1.7, 2.4, 5.2
+`mariadb` | 10.0 to 10.2  | 10.0 to 10.2
+`nginx`   | 1.9.0         | 1.9.0
+`node`    | 6, 8, 10, 11  | 6, 8, 10, 11
+`redis`   | 3.2, 4.0, 5.0 | 3.2, 4.0, 5.0
+`php`     | 7.1, 7.2      | Magento 2.2.5 and later—7.0, 7.1<br>Magento 2.2.4 and earlier—7.0.2, 7.0.4, 7.0.6, 7.1.0
+`rabbitmq`| 3.5, 3.7      | 3.5
+`varnish` | 4.0, 5.0      | 4.0, 5.0
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-conf-files_services.md]: Curious about the order of services in the list.  In the Launch Docker topic, the services are listed in alpha-order.

Comment (hguthrie) [reply guides/v2.2/cloud/project/project-conf-files_services.md]: You know, these should be also. I think all the table maneuvering has left it a little messed up. Will fix.

**Pull request merged**